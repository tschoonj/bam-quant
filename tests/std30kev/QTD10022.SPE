$DATA:
0 2000
23153
14896
6457
1829
383
55
10
0
1
0
0
0
0
0
0
1
1
28
16
23
22
23
15
23
20
15
18
19
18
20
13
13
16
15
15
20
21
11
21
22
30
20
20
17
12
12
14
11
13
7
11
13
11
16
16
12
10
15
10
17
9
10
18
20
11
11
9
16
10
9
13
15
6
11
10
16
9
13
11
12
11
17
14
10
12
17
14
12
14
13
10
18
5
14
11
10
15
14
15
15
11
16
9
17
15
17
14
15
19
8
9
11
11
11
5
12
14
15
20
5
10
13
12
10
11
11
12
13
12
14
12
10
9
14
13
11
12
15
12
10
11
14
6
10
6
17
13
14
19
8
13
21
7
13
6
6
11
12
16
20
17
12
19
7
14
10
10
13
11
14
15
7
10
6
9
12
6
12
9
16
14
15
15
12
8
8
9
13
8
7
6
10
12
7
6
6
6
2
5
10
3
6
11
10
8
6
2
4
10
5
8
7
8
5
3
4
3
7
5
2
3
4
4
0
1
4
2
3
3
3
2
4
5
5
3
3
2
2
5
3
2
2
2
2
4
1
1
6
2
2
4
0
2
3
1
1
2
3
0
1
3
1
3
0
0
3
4
1
3
3
4
4
3
2
3
4
2
0
2
1
2
2
4
2
3
1
3
4
1
1
2
3
0
1
1
1
3
1
1
0
1
2
0
4
0
2
2
1
2
3
1
1
3
2
1
6
9
3
6
4
1
1
2
3
3
1
2
1
1
1
1
2
1
1
2
2
1
0
0
0
1
4
1
4
1
0
3
1
3
0
3
1
0
0
0
0
0
3
3
1
0
2
2
2
0
1
1
1
0
0
0
0
1
1
2
2
0
2
2
1
1
2
0
0
0
1
2
0
0
1
1
0
1
0
2
0
2
1
0
0
1
3
0
0
1
0
0
0
0
1
0
1
0
0
0
1
0
3
1
2
2
0
2
2
1
1
2
0
0
0
0
0
1
0
0
1
1
0
1
0
1
1
2
2
0
1
0
1
1
0
0
2
0
1
0
1
3
0
0
1
2
0
1
0
2
1
2
1
1
0
1
0
1
0
1
0
2
1
0
1
0
1
2
0
0
0
0
0
2
0
2
0
3
1
0
0
4
2
0
2
0
2
1
1
1
0
1
0
2
0
1
1
0
0
2
0
0
0
1
0
1
3
1
1
3
2
2
3
2
1
1
2
3
0
2
3
2
2
0
1
2
2
0
1
0
2
1
0
1
1
0
0
1
3
2
3
1
1
2
1
1
2
1
1
1
0
0
1
2
1
0
2
3
5
1
3
1
2
0
1
4
1
4
2
1
3
5
2
0
5
1
3
4
2
2
0
1
2
0
2
2
3
7
1
2
4
5
3
0
3
1
2
2
3
0
4
0
1
5
2
1
1
3
3
3
2
5
4
1
2
1
3
3
1
2
3
3
0
5
2
1
5
1
3
5
4
2
0
3
3
2
3
3
1
1
3
1
2
3
2
4
3
3
5
1
1
0
2
1
2
2
1
2
2
4
1
3
4
2
5
3
4
4
3
0
4
5
3
2
3
3
5
4
3
4
4
4
4
3
3
4
2
6
1
1
6
3
4
2
2
5
3
0
7
3
4
4
8
4
3
8
1
4
1
3
3
6
3
4
3
7
5
2
1
6
0
3
3
7
7
2
6
4
3
3
8
4
5
5
6
5
4
4
3
3
4
5
5
1
5
5
5
4
2
3
9
10
8
8
14
24
36
44
60
82
84
119
94
131
141
136
112
116
114
111
73
56
56
36
24
18
15
11
6
6
4
5
5
4
3
5
4
5
3
4
7
2
1
4
2
6
0
3
5
3
4
4
4
3
2
3
4
6
3
5
2
3
5
0
5
0
5
3
12
2
2
6
4
0
7
4
3
2
2
4
1
6
2
6
0
7
7
7
4
6
5
7
1
7
5
4
2
5
6
4
9
8
11
9
17
18
20
27
16
24
25
26
21
11
19
15
12
10
10
12
3
12
11
8
6
11
3
5
7
5
8
5
5
3
5
4
5
3
1
4
6
5
4
4
3
3
2
4
4
4
5
2
2
2
8
3
8
1
3
6
3
8
4
5
2
4
2
9
4
5
7
5
3
3
5
4
5
1
5
2
2
0
5
3
5
5
2
2
4
2
2
8
7
2
4
3
5
7
4
7
6
3
2
2
6
1
3
3
3
6
1
3
5
2
3
5
4
4
3
4
3
2
1
4
2
1
5
5
2
1
5
1
5
3
3
1
3
2
3
4
4
3
2
4
4
6
6
3
3
1
4
4
4
6
2
2
1
2
2
5
2
3
6
0
6
2
5
2
5
2
3
2
0
4
5
1
2
2
3
3
1
2
1
3
4
3
1
2
2
1
4
3
8
6
2
0
3
7
4
3
0
4
2
1
7
6
3
5
1
1
1
4
3
2
2
3
2
4
5
3
4
5
6
1
2
3
2
4
5
8
2
6
1
1
5
4
5
6
3
3
3
6
5
2
2
3
2
1
0
4
8
2
4
4
2
3
3
0
3
4
1
4
4
2
3
3
3
3
2
5
5
1
4
5
4
1
4
1
6
3
8
3
1
4
0
5
1
1
1
3
4
1
8
4
2
0
1
6
2
3
3
3
5
3
1
4
1
1
1
2
5
3
6
3
2
3
5
5
1
4
4
8
1
1
3
1
1
3
5
2
4
1
3
2
1
4
2
7
4
4
4
5
2
2
1
4
1
2
3
5
3
5
3
2
5
1
3
1
2
6
8
1
7
6
4
5
8
5
9
8
11
16
11
11
12
15
14
8
14
22
10
20
24
21
20
26
24
18
25
24
16
14
10
8
4
8
5
5
6
6
10
3
6
2
3
4
2
4
2
3
8
3
0
2
7
1
5
7
3
5
3
6
2
2
8
7
8
12
8
7
8
7
10
10
9
4
9
6
10
8
7
6
5
10
10
4
6
8
6
3
6
7
2
3
3
2
2
7
2
0
2
4
5
9
5
7
3
4
4
2
5
5
4
5
2
5
5
8
5
4
9
10
5
10
7
13
6
3
10
8
8
4
8
8
3
6
9
9
1
9
9
10
14
10
9
6
4
6
11
8
14
9
7
7
4
5
7
7
6
5
9
8
3
5
6
8
6
8
5
4
5
5
5
8
11
9
5
11
8
9
11
9
8
22
13
16
13
18
13
13
11
14
15
26
12
9
11
12
11
10
18
12
7
10
14
11
13
12
10
12
16
13
16
12
11
11
8
16
13
12
10
15
18
7
19
11
18
14
17
13
9
14
13
15
17
16
19
18
18
19
15
12
11
13
21
13
18
11
15
15
12
15
20
11
15
19
15
13
9
17
15
14
15
14
23
11
19
17
10
14
10
12
21
19
15
26
11
9
16
15
21
19
19
16
10
11
11
16
13
21
22
16
18
17
21
22
29
15
22
17
25
25
23
18
18
24
16
23
18
28
27
30
27
36
33
27
28
32
28
20
29
25
34
23
28
30
32
30
39
25
33
27
22
31
34
28
24
36
27
33
37
28
31
48
39
35
38
35
38
46
43
42
41
35
47
40
59
46
32
44
50
62
63
59
42
58
72
56
54
84
54
69
60
55
59
51
69
65
79
76
70
77
81
72
69
83
78
69
87
93
104
110
96
92
103
91
83
95
87
114
97
88
93
79
96
99
101
103
82
92
83
76
94
76
65
89
75
81
59
82
75
78
71
67
71
75
70
64
58
54
76
55
71
62
58
68
56
65
47
48
49
47
55
56
52
55
56
39
50
38
60
41
50
45
37
49
43
44
28
33
50
42
46
51
41
42
59
25
39
41
48
39
32
37
31
34
47
36
25
41
29
32
34
25
26
23
37
28
31
36
28
42
42
46
37
56
38
59
42
42
52
84
68
75
73
70
79
77
74
85
84
97
76
93
79
78
89
84
81
66
62
76
71
62
63
53
71
52
45
48
54
47
32
29
29
24
22
17
19
18
9
9
12
4
3
5
2
3
2
2
0
2
1
1
0
0
0
1
0
0
0
0
0
0
1
0
0
0
2
0
0
0
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10022.spe
Start: 22.07.2014 09:54:04
Stop: 22.07.2014 09:55:44
BESSY: 298.1574785 298.2649863
IONI: 2.074998E-11 2.054534E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98072
Run: 100000
Dead: 2
ICR: 240
OCR: 0
Accumulated BESSY: 59713.1292218
Accumulated Ioni: 4.14065051E-09
N Accumulation: 200
Normfactor:  4.9251151e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ti
 
exp_x 72.7203979492188 72.7204
exp_y 150.465789794922 150.4658
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  22   1      4.509         1.         5.       1.25
