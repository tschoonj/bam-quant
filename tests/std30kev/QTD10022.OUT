     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:42  บ   
     บ  Spectrum: QTD10022.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        2 iterations done    บ   
     บ  ChiSquare =    10.3   last change =   .00%    lambda= 1.E-04   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -52.5 ๑   100.0   บ     -52.5 ๑    99.0   บ   
     บ  GAIN (eV/ch)  บ     20.150 ๑   2.000   บ    20.150 ๑   1.627   บ   
     บ det NOISE (eV) บ      193.8 ๑    40.0   บ     193.8 ๑    63.0   บ   
     บ  FANO factor   บ       .035 ๑    .050   บ      .035 ๑    .050   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Ti-Ka                                 1. ๑      5.           บ   
     บ    KA1         4.509   1.00000           1. ๑      8.      1.3  บ   
     บ              226.375    202.43          35.           7.84E-01  บ   
     บ                                                                 บ   
     บ  2 Ti-Kb                                 0. ๑      5.           บ   
     บ    KB1         4.932   1.00000           0. ๑      8.       .8  บ   
     บ              247.367    203.22          31.           8.30E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
